graphene_django/views.py

```diff
@@ -55,6 +56,11 @@ def instantiate_middleware(middlewares):
         yield middleware
 
 
+def stringify_datetime(value):
+    if isinstance(value, datetime.datetime):
+        return value.__str__()
```

Comment:
what happens if the value is not a datetime instance? shouldn't we just try to stringify everything?

```suggestion
def stringify(value):
    return str(value)
```